k8s.tf

```diff
@@ -38,7 +38,18 @@ module "eks-addons" {
     dependency_update = true
   }
 
-  cluster_autoscaler_helm_config = var.cluster_autoscaler_helm_config
+  ingress_nginx_helm_config = {

```

Comment:
Did you add this `ingress_nginx_helm_config` intentionally?

Reply:
No, this was a failure from manual merge conflict resolution. Removed it.